orangecontrib/explain/widgets/owice.py

```diff
@@ -600,12 +600,9 @@ def __on_target_changed(self):
         self._apply_feature_sorting()
         self.__on_parameter_changed()
 
-    def __on_feature_changed(self, selection: QItemSelection):
-        if not selection:
-            return
-
-        self.feature = selection.indexes()[0].data(gui.TableVariable)
-        self._apply_feature_sorting()
+    def __on_feature_changed(self):
+        selection = self._features_view.selectionModel().selectedIndexes()
```

Comment:
Sometimes while filtering, `__on_feature_changed` is called two times. First with a selection change and second without change (both selected and deselected parameters are empty). Since it is safer, use `self._features_view.selectionModel().selectedIndexes()`.